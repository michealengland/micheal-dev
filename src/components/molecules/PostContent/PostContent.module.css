.postContent {
  & .inner {
    @apply p-4 max-w-3xl mx-auto;
  }

  & h1 {
    @apply text-3xl mb-4;
  }

  & h2 {
    @apply text-2xl mb-4;
  }

  & h3 {
    @apply text-xl mb-4;
  }

  & h4 {
    @apply text-lg;
  }

  & p {
    @apply mb-6;

    & code {
      @apply text-primary-purple1 bg-purple-100 font-mono text-sm rounded-sm;

      padding: 0.125rem;
    }
  }

  & pre {
    @apply text-lg mb-4 rounded-sm py-4 font-mono;
  }
}
