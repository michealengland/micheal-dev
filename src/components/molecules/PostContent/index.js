import React from 'react'
import PropTypes from 'prop-types'

export default function PostContent({children, className}) {
  return <section className={className}>{children}</section>
}

PostContent.propTypes = {
  children: PropTypes.node,
  className: PropTypes.string
}
