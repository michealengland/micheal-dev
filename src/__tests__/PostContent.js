import React from 'react'
import renderer from 'react-test-renderer'
import PostContent from '@components/molecules/PostContent'
describe('PostContent', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <PostContent className="post-content">
        <p>Test</p>
      </PostContent>
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
