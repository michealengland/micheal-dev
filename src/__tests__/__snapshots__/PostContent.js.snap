// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PostContent renders correctly 1`] = `
<section
  className="postContent post-content"
>
  <div
    className="inner"
  >
    <p>
      Test
    </p>
  </div>
</section>
`;
